iambic/config/wizard.py

```diff
@@ -1964,8 +1996,10 @@ def configuration_github_app_aws_lambda_setup(self):
         }
         available_account_names = sorted(list(account_name_to_account_id.keys()))
         questionary_params = {}
-        question_text = "We recommend you deploy Lambda integration on a non-management account.\nTarget AWS Account name: "
-        if len(available_account_names) < 10:
+        question_text = "We recommend you deploy Lambda integration on a non-management account.\nTarget AWS Account profile: "
```

Comment:
not sure if profile is the right term. they are currently really names from the organization

Reply:
OH you are right! I swear I saw a profile name there. This is why (for internal only): https://github.com/iambic-test-org/iambic-templates/blob/main/iambic_config.yaml#L6